third_party/3/waitress/adjustments.pyi

```diff
@@ -0,0 +1,60 @@
+from .compat import HAS_IPV6, PY2, WIN, string_types
+from .proxy_headers import PROXY_HEADERS
+from socket import SocketType
+from typing import Any, Mapping, Sequence, Optional, Set, Tuple, Union
+
+truthy: frozenset
+KNOWN_PROXY_HEADERS: frozenset
+
+def asbool(s: Optional[Union[bool, str, int]]) -> bool: ...
+def asoctal(s: str) -> int: ...
+def aslist_cronly(value: str) -> Sequence[str]: ...
+def aslist(value: str) -> Sequence[str]: ...
+def asset(value: str) -> Set[str]: ...
+def slash_fixed_str(s: str) -> str: ...
+def str_iftruthy(s: str) -> Optional[str]: ...
+def as_socket_list(sockets: Sequence[Any]) -> Sequence[SocketType]: ...
```

Comment:
`Any` can be `object` here as it can be literally anything. In arguments, `Any` is more a marker for "can't be expressed properly using the type system".